packages/components/src/font-size-picker/font-size-picker-toggle-group.tsx

```diff
@@ -19,6 +19,7 @@ const FontSizePickerToggleGroup = ( props: FontSizePickerToggleGroupProps ) => {
 		<ToggleGroupControl
 			__nextHasNoMarginBottom
 			__next40pxDefaultSize={ __next40pxDefaultSize }
+			__shouldNotWarnDeprecated36pxSize
```

Comment:
I think we're also going to need a prop like this to prevent the redundant warnings when used inside another component: 1b7eeec

Reply:
Hi @mirka should we also update the usage of `ToggleGroupControl` by adding this prop in other packages as well like block editor and library?

Reply:
Not that I'm aware of. We only want to suppress the warning when ToggleGroupControl's size is dependent on a parent component's size, and the parent component is responsible for logging the warning. I'm not aware of any other cases like that.